assets/src/dashboard/utils/useTemplateView.js

```diff
@@ -35,6 +35,7 @@ export default function useTemplateView({ totalPages }) {
   const [searchKeyword, _setSearchKeyword] = useState('');
   const [sort, _setSort] = useState(TEMPLATES_GALLERY_SORT_OPTIONS.POPULAR);
   const [page, setPage] = useState(1);
+  const [preview, setPreview] = useState();
```

Comment:
Is the idea this would store the AMP markup for the story/template to be previewed?

Reply:
i am thinking that the template or the template id gets passed to the preview, to basically communicate 'current preview is x template' and then we would take the `preview` and then render the AMP markup in a separate hook so that it is a neutral place for templates and stories and use that in the preview itself. But this is more of just a flag for the current view state. 

thoughts?

Reply:
Yeah, I think that makes sense, perhaps it could be named if it's a flag for the view state as `setPreviewVisible`